docs/how_to/enabling_access_to_vmpooler.md

```diff
@@ -0,0 +1,18 @@
+# Enabling access to the vmpooler VMs during an acceptance test run
+
+If you are running acceptance tests for Beaker that utilise vmpooler, you will need to run an SSH agent and add the acceptance RSA key prior to running the tests
+
+To load the SSH agent and add the acceptance key, run the following:
+
+~~~bash
+eval `ssh-agent`
+ssh-add ~/.ssh/id_rsa-acceptance
```

Comment:
I noticed that some usages in ci-job-configs also add the `id_rsa` key.

I'm wondering if both are necessary in some cases?

Is the `id_rsa` used for cloning private repos from GitHub, and the `id_rsa-acceptance` used for SSH'ing into VMPooler VMs?